src/MvvmNano.Forms/NanoMasterDetailPage.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System.Linq; 
+using Xamarin.Forms;
+
+namespace MvvmNano.Forms
+{
+    /// <summary>
+    /// A MasterDetailPage implementation that fits to the MvvmNano framework.
+    /// </summary>
+    public class NanoMasterDetailPage :  MasterDetailPage
```

Comment:
I had some trouble casting MainPage to MvvmNanoMasterDetailPage<MvvmNanoViewModel> in the MvvmNanoMasterDetailApplication and that was my way to fix it.

-> 

       ' /// <summary>
        /// The <see cref="NanoMasterDetailPage"/> if one is set. 
        /// </summary>
        public NanoMasterDetailPage MasterPage => MainPage as NanoMasterDetailPage;`

I will take a look into it again and check if i can resolve it in a more convinient way.